CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ command line for details.
 
 
 ## [Unreleased]
+- Use GITLAB_URL instead of GITLAB_HOST to specify URL of your GitLab instance.
```

Comment:
I think it's ok to add GITLAB_URL but let's leave GITLAB_HOST as it currently is. According to the [GITLAB docs](https://docs.gitlab.com/ee/administration/environment_variables.html#doc-nav), GITLAB_HOST is the recommended environment variable and should include "The full URL of the GitLab server (including http:// or https://)"

Reply:
I might submit a PR also to GitLab too. It is really confusing to call a variable `HOST` if it **must** contain an URL.

Reply:
Let's do this to minimize user confusion. State in the release notes that the full URL should be set in both GITLAB_URL and GITLAB_HOST (current GitLab documentation).

We should not redefine what GITLAB_HOST means in this PR.

Reply:
Let's start with simply by recommending correct naming. The `GITLAB_HOST` will work and it will warn developers that use common sense and set service hostname to variable that it is called `_HOST`.